isort/stdlibs/py310.py

```diff
@@ -6,10 +6,105 @@
 """
 
 stdlib = {
+    "__hello__",
+    "__phello__",
+    "_abc",
+    "_aix_support",
     "_ast",
+    "_asyncio",
+    "_bisect",
+    "_blake2",
+    "_bootsubprocess",
+    "_bz2",
+    "_codecs",
+    "_codecs_cn",
+    "_codecs_hk",
+    "_codecs_iso2022",
+    "_codecs_jp",
+    "_codecs_kr",
+    "_codecs_tw",
+    "_collections",
+    "_collections_abc",
+    "_compat_pickle",
+    "_compression",
+    "_contextvars",
+    "_crypt",
+    "_csv",
+    "_ctypes",
+    "_ctypes_test",
+    "_curses",
+    "_curses_panel",
+    "_datetime",
+    "_dbm",
+    "_decimal",
+    "_elementtree",
+    "_frozen_importlib",
+    "_frozen_importlib_external",
+    "_functools",
+    "_gdbm",
+    "_hashlib",
+    "_heapq",
+    "_imp",
+    "_io",
+    "_json",
+    "_locale",
+    "_lsprof",
+    "_lzma",
+    "_markupbase",
+    "_md5",
+    "_msi",
+    "_multibytecodec",
+    "_multiprocessing",
+    "_opcode",
+    "_operator",
+    "_osx_support",
+    "_overlapped",
+    "_pickle",
+    "_posixshmem",
+    "_posixsubprocess",
+    "_py_abc",
+    "_pydecimal",
+    "_pyio",
+    "_queue",
+    "_random",
+    "_scproxy",
+    "_sha1",
+    "_sha256",
+    "_sha3",
+    "_sha512",
+    "_signal",
+    "_sitebuiltins",
+    "_socket",
+    "_sqlite3",
+    "_sre",
+    "_ssl",
+    "_stat",
+    "_statistics",
+    "_string",
+    "_strptime",
+    "_struct",
+    "_symtable",
+    "_testbuffer",
+    "_testcapi",
+    "_testclinic",
+    "_testconsole",
+    "_testimportmultiple",
+    "_testinternalcapi",
+    "_testmultiphase",
+    "_thread",
+    "_threading_local",
+    "_tkinter",
+    "_tracemalloc",
+    "_uuid",
+    "_warnings",
+    "_weakref",
+    "_weakrefset",
+    "_winapi",
+    "_xxsubinterpreters",
+    "_xxtestfuzz",
+    "_zoneinfo",
     "abc",
     "aifc",
-    "antigravity",
```

Comment:
I would prefer not to remove `antigravity` and other modules that are shipped with python.. they are valid anyway.

Reply:
Done.